src/apps/setup.ts

```diff
@@ -52,7 +52,10 @@ export const setupAppFactory = (
 
     route.get("/probot/setup", async (req: Request, res: Response) => {
       const { code } = req.query;
-      const response = await setup.createAppFromCode(code);
+      const response = await setup.createAppFromCode(code, {
+        // @ts-ignore
+        request: app.state.request,
```

Comment:
state is private. make it public?

Reply:
no I think that's okay as a workaround. I would love to move the whole logic of `createAppFromCode` into its own separate module

Reply:
that should be part of a following PR :)